deploy/tutorials/tutorial-firebase.md

```diff
@@ -311,30 +311,36 @@ app.use(async (ctx, next) => {
 Now let's add our router to the middleware application and set the application
 to listen on port 8000:
 
-```js
+```js title="firebase.js"
 app.use(router.routes());
 app.use(router.allowedMethods());
 await app.listen({ port: 8000 });
 ```
 
 Now we have an application that should serve up our APIs.
 
-## Create a Project in Deno Deploy
+## Deploy the Application
 
-1. Go to [https://dash.deno.com/new](https://dash.deno.com/new) (Sign in with
-   GitHub if you didn't already) and click on **+ Empty Project** under **Deploy
-   from the command line**.
-2. Now click on **Settings** button available on the project page.
-3. Navigate to **Environment Variables** Section and add the following:
+Now that we have everything in place, let's deploy your new application!
 
-   <dl>
-    <dt><code>FIREBASE_USERNAME</code></dt>
-    <dd>The Firebase user (email address) that was added above.</dd>
-    <dt><code>FIREBASE_PASSWORD</code></dt>
-    <dd>The Firebase user password that was added above.</dd>
-    <dt><code>FIREBASE_CONFIG</code></dt>
-    <dd>The configuration of the Firebase application as a JSON string.</dd>
-   </dl>
+1. In your browser, visit [Deno Deploy](https://dash.deno.com/new_project) and
+   link your GitHub account.
+2. Select the repository which contains your new application.
+3. You can give your project a name or allow Deno to generate one for you
+4. Select `firebase.js` in the Entrypoint dropdown
+5. Click **Deploy Project**
+
+In order for your Application to work, we will need to configure its environment
+variables.
+
+On your project's success page, or in your project dashboard, click on **Add
+environmental variables**. Under Environment Variables, click **+ Add
```

Comment:
A bit tangential topic: We usually refer to it as environment variables (not environment_al_) so I found this expression a bit weird - we might want to fix the success page to use "environment" here

Reply:
Will make an issue for it